minifi-docker/dockerhub/Dockerfile

```diff
@@ -34,6 +34,8 @@ RUN mkdir -p $MINIFI_HOME
 
 RUN apk --no-cache add curl
 
+ADD sh/ ${MINIFI_BASE_DIR}/scripts/
```

Comment:
Noticed below it's used as just `$MINIFI_BASE_DIR`. Is the `{}` needed?

Reply:
Either format is valid, and in this case, equivalent. 

From the [POSIX spec](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02): 

> The simplest form for parameter expansion is:
>
>    ${parameter}
>
> The value, if any, of parameter shall be substituted.
> 
> The parameter name or symbol can be enclosed in braces, which are optional... If the parameter is not enclosed in braces, and is a name, the expansion shall use the longest valid name

Reply:
Wondering more from a consistency aspect but I don't think it matters. Thanks!

Reply:
Fair point. It also occurred to me later that I'm not sure if Dockerfile syntax has any special interpretation / side-effect that differs for POSIX... of that I'm not positive but in any case it seems to work ok :)

Reply:
I know I can be a nitpicker sometimes on pull requests just to try to bring stuff to attention to make sure it's what was intended. :) I have used `$VAR` but I don't think I have ever used `${VAR}` in a Dockerfile.

Reply:
I was curious and looked it up as well. I found the correct answer for Dockerfile syntax here: https://docs.docker.com/engine/reference/builder/#environment-replacement

> Environment variables are notated in the Dockerfile either with $variable_name or ${variable_name}. They are treated equivalently and the brace syntax is typically used to address issues with variable names with no whitespace, like ${foo}_bar

Good eye, and I agree it's always better to point something out on reviews. Everyone sees things slightly differently, and even if "correct" functionally, it may not be the author's intention. Thanks!